File: components/InstructionScreen.js
import React from 'react';
import { View, Text, StyleSheet, ScrollView } from 'react-native';

export default function InstructionScreen() {
  return (
    <ScrollView contentContainerStyle={styles.container}>
      <Text style={styles.heading}>How to Use the App</Text>
      <Text style={styles.instruction}>Add new jaap counting section.</Text>
      <Text style={styles.instruction}>Navigate to the section you want to use for jaap counting.</Text>
      <Text style={styles.instruction}>Click on the count button to increase the count.</Text>
      <Text style={styles.instruction}>Use the clear button to reset the count.</Text>
      {/* Add more instructions as needed */}
    </ScrollView>
  );
}

const styles = StyleSheet.create({
  container: {
    padding: 16,
  },
  heading: {
    fontSize: 24,
    fontWeight: 'bold',
    marginBottom: 20,
  },
  instruction: {
    fontSize: 18,
    marginBottom: 10,
    
  },
});

File: lib/screens/instruction_screen.dart
import 'package:flutter/material.dart';
import 'package:shared_preferences/shared_preferences.dart';

class InstructionScreen extends StatelessWidget {
  @override
  Widget build(BuildContext context) {
    return Scaffold(
      body: Padding(
        padding: EdgeInsets.all(16),
        child: ListView(
          children: [
            Text('How to Use the App'),
            Text('Add new jaap counting section.'),
            // More instructions as needed...
          ],
        ),
      ),
    );
  }
}
